notebook-lm-clone/src/podcast/script_generator.py

```
@@ -0,0 +1,299 @@
+        return json.dumps({
+            'script': self.script,
+            'metadata': {
+                'source_document': self.source_document,
+                'total_lines': self.total_lines,
+                'estimated_duration': self.estimated_duration
+            }
+        }, indent=2)
+
+
+class PodcastScriptGenerator:
+    def __init__(self, openai_api_key: str, model_name: str = "gpt-4o-mini"):
```

Ruff (0.13.1)

Unused method argument: `openai_api_key`

(ARG002)